FIAT/__init__.py

```diff
@@ -30,6 +30,9 @@
 from FIAT.crouzeix_raviart import CrouzeixRaviart
 from FIAT.regge import Regge
 from FIAT.hellan_herrmann_johnson import HellanHerrmannJohnson
+from FIAT.arnold_winther import ArnoldWinther
+from FIAT.arnold_winther import ArnoldWintherNC
+from FIAT.mardal_tai_winther import MardalTaiWinther  # noqa:F401
```

Comment:
So why is it that AW elements are being added to the `dict`, but MTW is not?

Reply:
oversight.  Fixed